packages/components/pie-card-container/test/component/pie-card-container.spec.ts

```diff
@@ -229,4 +229,126 @@ test.describe('PieCardContainer - Component tests', () => {
             });
         });
     });
+
+    test.describe('Prop: `padding`', () => {
+        test.describe('when `padding` is set as a single string value', () => {
+            test('should set an attribute of style with the correct padding value', async ({ mount, page }) => {
+                // Arrange
+                await mount(PieCardContainer, {
+                    props: {
+                        padding: 'a',
+                    } as CardContainerProps,
+                    slots: {
+                        default: slotContent,
+                    },
+                });
+
+                const component = page.locator(componentSelector);
+                const componentAttribute = await component.getAttribute('style');
+
+                // Assert
+                expect(componentAttribute).toBe('padding: var(--dt-spacing-a)');
+            });
+
+            padding.forEach((padding) => {
```

Comment:
Variable name clash here, surprised it's not causing any issues?

Reply:
Fixed.